v22.1/vault-db-secrets-tutorial.md

```diff
@@ -0,0 +1,472 @@
+  v-token-hc-crdb-rol-wltbv25napuroomvzmok-1653515524 | VALID UNTIL=2022-05-25 22:52:09+00:00 | {}
+~~~
+
+### Revoke credentials
+
+Revoke a lease or two just for the feeling of raw power. Recall that a "lease" on a "role" in Vault terms corresponds to an actual credential pair on the CockroachDB cluster, i.e., a username/password combination for a SQL user.
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+vault lease revoke database/creds/crdb-role/XIpIBRM8FkQxD5B0ndB1lszY.iMXtW
+~~~
+
+~~~txt
+All revocation operations queued successfully!
+~~~
+
+### Provision a CockroachDB-client Vault policy
+
+As admin, provision a Vault policy for CockroachDB client operators to access the client credentials.
+
+The purpose of the previous work is to make a dynamic secret that can be used to access the CockroachDB database by generating credentials on demand. Performing the above work (establishing the connection between the CockroachDB cluster and the Vault cluster, creating the template for database client credentials, etc.) required admin privileges. But for the work to be meaningful, a Vault user with more limited permisions must be able to access the generated credentials.
+
+This policy will be used to access CockroachDB client credentials. In keeping with the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege), let's give it only the required ability to read the required credential. 
+
+Vault policies are specified using HashiCorp Configuration Language (HCL). The following configuration specifies a policy of read access for the `crdb-role` credential.
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+vault policy write roach-client - <<hcl
+path "database/creds/crdb-role" {
+  capabilities = [ "read" ]
+}
+hcl
+~~~
+
+~~~shell
+Success! Uploaded policy: roach-client
+~~~
+
+### Generate an authentication token for the `crdb-role` Vault user
+
+Our final act as Vault admin will be to provision an authentication token to assume the `crdb-role`, which represents the Vault role with the sole purpose of providing credentials ot a CockroachDB client.
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+ vault token create -policy=roach-client
+~~~
+
+~~~txt
+Key                  Value
+---                  -----
+token                hvs.CAESIK4TK7JcSJuKxOgwEa3mYhvfN356Uhikij821K4E4XnWGigKImh2cy5MeUJ5dGNxMjRzNk9qNmVDYWtFYjRUd2QuaU1YdFcQ1cAT
+token_accessor       36udxC5m0hmA0niBYyCThk0k.iMXtW
+token_duration       1h
+token_renewable      true
+token_policies       ["default" "roach-client"]
+identity_policies    []
+policies             ["default" "roach-client"]
+~~~
+
+You can either copy the `token` from the output of the previous command, or capture a token with the following command.
+
+~~~shell
+ VAULT_CLIENT_TOKEN=`vault token create -policy=roach-client -format=json | jq .auth.client_token | tr -d '"'`
+~~~
+
+## Connecting to CockroachDB with Vault-provisioned credentials
```

Comment:
Between this main heading "Connecting to..." and the H3 heading, do you think it would be worth having just a contextual couple sentences to tell the reader/user what they're about to do. That seems to be what you have done for the other steps.